### ISSUE TITLE
[Feature Request] Open app from launcher into new workspace

### ISSUE THREAD

**Issue by alaurie** [pop-os/shell#361]
It would be great if we could launch and app from the launcher into a new empty desktop/workspace rather than the one you are currently in. Possibly a sub option in the menu or another keyboard shortcut.

**Comment by Jom97** [comment]
yeah absolutely agree. this is something you're constantly doing. So it makes sense for there to be an easier way to do this. But i guess this is more of an upstream issue than a pop os one. Even something like a keyboard shortcut to switch to the bottom-most workspace would make things a lot smoother.

**Comment by sobeitnow0** [comment]
Not quite what yout wanna but:Move applications to specific workspaces when they create windows. 


https://extensions.gnome.org/extension/16/auto-move-windows/
